2019-Container/ProjectSettings/AndroidResolverDependencies.xml

```diff
@@ -29,7 +29,7 @@
     <setting name="packageDir" value="Assets/Plugins/Android" />
     <setting name="patchAndroidManifest" value="True" />
     <setting name="patchMainTemplateGradle" value="True" />
-    <setting name="projectExportEnabled" value="True" />
+    <setting name="projectExportEnabled" value="False" />
```

Comment:
You can rollback this

Reply:
Will do, I don't know why Unity does this auto-generated.

Reply:
Done!